examples/node/index.js

```diff
@@ -135,6 +138,7 @@ function queryUserInfo(parentSpan, username, callback) {
         });
 
         // In parallel, query the recent events activity for the user
+        console.log(json.received_events_url);
```

Comment:
Just checking: is this intentional or a debug statement that didn't get removed?